aten/src/ATen/core/Tensor.cpp

```diff
@@ -28,7 +29,7 @@ TensorBase TensorBase::__dispatch_contiguous(c10::MemoryFormat memory_format) co
 
```

Comment:
It would be weird to not cover `contiguous` and `zero_` if we are going to change `fill_`. I suggest let's add tests to cover for these two ops as well.